register/models.py

```diff
@@ -0,0 +1,22 @@
+from django.utils import timezone
+from django.db import models
+from secrets import token_urlsafe
+
+
+class Token(models.Model):
+    code = models.CharField(max_length=43)
+    expiry = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
+        if not self.pk:
+            self.code = token_urlsafe()
+            self.expiry = timezone.now() + timezone.timedelta(days=1)
```

Comment:
I don't feel comfortable with these two lines. I mean, when you create a new token like `t = Token()` you are creating an empty object and the valuers are set when you save it. I think the values should be set on the constructor. 
We can talk about it